modules/unsupported/geojsonstore/src/main/java/org/geotools/data/geojson/GeoJSONWriter.java

```diff
@@ -57,6 +68,14 @@
 public class GeoJSONWriter implements AutoCloseable {
     static Logger LOGGER = Logging.getLogger("org.geotools.data.geojson");
 
+    /** Date format (ISO 8601) */
+    public static final String DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSSZ";
+
+    public static final TimeZone TIME_ZONE = TimeZone.getTimeZone("GMT");
```

Comment:
Does GeoJSON have to be in GMT? or is this something we should let the user set/override?

Reply:
Hum... not sure. JSON per se is not mandating any representation, and while ISO is the most recommended format, even for interoperability, I see no mention of timezone, though the examples normally come in GMT, e.g.:
- https://stackoverflow.com/questions/10286204/what-is-the-right-json-date-format
- https://tools.ietf.org/html/rfc7493

Maybe it could be configurable, but default to GMT?

Reply:
I would say default to GMT but allow people to override is probably best and possibly allow them to override the date format too?